src/Ant/Checkbox/Css.elm

```diff
@@ -26,14 +34,33 @@ emptyContent : Style
 emptyContent =
     property "content" "\"\""
 
+withFocus : String -> String
+withFocus class = class ++ ":focus"
+
+withHover : String -> String
+withHover class = class ++ ":hover"
+
+withChecked: String -> String
+withChecked class = class ++ "-checked"
+
+withAfter : String -> String
+withAfter class = class ++ "::after"
+
+childrenSelectorString : List String -> String
+childrenSelectorString l =
+    String.join " " (List.map (String.append ".") l)
```

Comment:
This function creates a descendants selector, not children selector. The depth is arbitrary here.

Please rename to be called `descendantsSelectorString`.